src/models/partial_tx.ts

```diff
@@ -229,13 +210,12 @@ export class PartialTx {
     const tokens = Array.from(tokenSet);
 
     const data = {
+      version: DEFAULT_TX_VERSION,
       tokens,
       inputs: this.inputs.map(i => i.toData()),
       outputs: this.outputs.map(o => o.toData(tokens.indexOf(o.token) + 1, this.network)),
     };
 
-    transaction.completeTx(data);
```

Comment:
Yes, because the returned `IDataTx` will be used to create a `Transaction` instance, which can call its `prepareToSend` method which adds the required fields.